.travis.yml

```diff
@@ -1,9 +1,10 @@
 language: node_js
 node_js:
-  - '8'
+  - 10
 before_install:
   - rm -rf build
   - rm -rf node_modules
+  - npm install -g npm@6.13
```

Comment:
Do you know why this is needed?

Reply:
Because or else it uses the base NPM version released with NodeJS 10 which is older. I think if we want consistent lock files we need to enforce specific NPM versions as well (since the lock format changes in minor ways with different versions)

It’s not technically needed, but it’s nice to document an NPM version that we know works.